app/models/device.rb

```diff
@@ -41,7 +41,8 @@ def ios_notification(title, body, payload)
           body: body.truncate(512)
         },
         "thread-id": Settings::Community.community_name,
-        sound: "default"
+        sound: "default",
+        "mutable-content": 1
```

Comment:
Do you maybe want to include a link to the Apple docs linked in the PR description here? It would be a shame for this context to get lost in a PR.

Reply:
.